cli.js

```diff
@@ -74,7 +74,12 @@ program
         var gitignore = fs.readFileSync('.gitignore', 'utf8');
 
         if (gitignore.indexOf('.gh-pages-cache') === -1) {
-          return promptly.confirm('.gh-pages-cache is a temporary repository that we use to push changes to your gh-pages branch. Do you want to add it to .gitignore?').then(function(answer) {
+          return promptly.confirm(chalk.yellow.bold(
+            '⚠ .gh-pages-cache is a temporary repository that I use to push\n' +
+            '  changes to your gh-pages branch. It isn\'t in your .gitignore file,\n' +
+            '  which means that Git will report it as an untracked file.\n\n' +
+            '  Do you want to add .gh-pages-cache to .gitignore (Y/n)?'), { default: true })
```

Comment:
I tried both and couldn't decide which one was better, I'll follow your advice: 3e97a2f24d1ed8638e919243e192cd6968f41c76